dts/bindings/sensor/invensense,icm42688.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2022 Intel Corporation
+# Copyright (c) 2022 Esco Medical ApS
+# Copyright (c) 2020 TDK Invensense
+# SPDX-License-Identifier: Apache-2.0
+
+description: ICM-42688 motion tracking device
+
+compatible: "invensense,icm42688"
+
+include: [sensor-device.yaml, spi-device.yaml]
+
+properties:
+    friendly-name:
+      default: "ICM42688"
```

Comment:
I don't think it makes sense to set a default friendly name in the binding; especially if that default is just the model name.

Reply:
Done